### PR TITLE
Event Editor 

### PR DESCRIPTION
**Pull request by barnagoz**
## Progress

- [x] #220 
- [x] #219 
- [x] #221
- [ ] #222 
- [x] #218 

Fixes: #223

### CODE REVIEW

**Comment by barnagoz** [comment]
A create event page gate-je a következő commitban meg lesz oldva.

**Comment by difcsi** [comment]
Build and test workflow broken on this branch: will be fixed after other PR is merged. Will admin bypass when merging

**Comment by barnagoz** [comment]
Should be ready for review. #222 will be done seperately

File: resources/frontend/pages/event/create.tsx [hunk 1]
```diff
@@ -177,4 +174,4 @@ const CreateEventPage = () => {
     );
 };
 
-export default gated(CreateEventPage, isOperator);
```
Comment (difcsi) [inline resources/frontend/pages/event/create.tsx:9]: Reverting this to isAdmin. We don't want fake event spam

File: resources/frontend/pages/event/[eventid]/manage/edit.tsx [hunk 2]
```diff
@@ -67,15 +69,15 @@ const initialValues: {
```
Comment (difcsi) [inline resources/frontend/pages/event/[eventid]/manage/edit.tsx:69]: Itt ez nem lesz így jó

**Review by difcsi — CHANGES_REQUESTED** [review]
Kicsit sok a duplikáció, meg néha a copypaste miatt lemaradt pár feature (event editor nem tölti be az értékeket)

**Comment by difcsi** [comment]
closing PR as it has been merged with other PR: CRUD boilerplate present there could make this way simpler

**Pull request closed without merging**